python/openai/openai_frontend/engine/triton_engine.py

```diff
@@ -250,6 +250,9 @@ def _get_model_metadata(self) -> Dict[str, TritonModelMetadata]:
         for name, _ in self.server.models().keys():
             model = self.server.model(name)
             backend = model.config()["backend"]
+            if backend == "":
+                # look for ensemble setting if backend is undefined
```

Comment:
```suggestion
            if not backend:
                # Check platform field as a backup, this will support 'ensemble' models
```

Reply:
This actually does not work. It returns an empty string, not None